### DISCUSSION TITLE
禁用guest账户，怎么取消Guest user is disabled, login please 提示

### DISCUSSION THREAD

**Discussion by Yumega** [AlistGo/alist#6995] [Q&A]
每次打开都有 Guest user is disabled, login please  这样的提示，怎么禁止提示
Thanks

**Comment by anwen-anyi** [comment]
不能访客查看 只能提醒登录了
或者你把访客打开，但是把所有内容都隐藏了，这样就不会提醒了

**Reply by Yumega** [reply]
> 不能访客查看 只能提醒登录了 或者你把访客打开，但是把所有内容都隐藏了，这样就不会提醒了

如何隐藏

**Reply by Simonzhang8** [reply]
> > 不能访客查看 只能提醒登录了 或者你把访客打开，但是把所有内容都隐藏了，这样就不会提醒了
> 
> 如何隐藏


guest用户默认文件夹设置成一个空文件夹
